CONTRIBUTING.md

```diff
@@ -122,6 +164,7 @@ changed and why. Follow these guidelines when writing one:
     Refs: http://eslint.org/docs/rules/space-in-parens.html
     Refs: https://github.com/nodejs/node/pull/3615
     ```
+    - It's it's not a fix, you should document the motivation for the change
```

Comment:
Fixed and removed `you`, but there are a ton of previously existing `your` in there.